infrastructure/serve/cosmos.tf

```diff
@@ -0,0 +1,73 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+resource "azurerm_cosmosdb_account" "serve" {
+  name                = "cosmos-db-${var.naming_suffix}"
+  location            = var.core_rg_location
+  resource_group_name = var.core_rg_name
+  offer_type          = "Standard"
+  kind                = "GlobalDocumentDB"
+  # public_network_access_enabled = var.local_mode
```

Comment:
off by default or wait for [damoo/feature-data-store](https://github.com/UCLH-Foundry/FlowEHR/tree/damoo/feature-data-store) to be merged?

Reply:
We'll likely also want to use this https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cosmosdb_account#local_authentication_disabled and make that configurable, set to true for prod

Reply:
As mentioned on IM perhaps we leave this commented and merge so it's not held up any longer?